script/verify.py

```diff
@@ -159,6 +133,36 @@ async def update_roles(self, ctx):
                 await self.give_roles(member, mem_entry)
         await ctx.send("Roles Updated")
 
+    async def authorize_user(self, ctx, logs, name, entry_number):
+        google_auth_service = GoogleAuthService()
+        auth_url = google_auth_service.auth_url()
+        verication_message = 'Please visit the following link for verification: {}'
+
+        await ctx.send(f'Name : {name}\nEntry Number : {entry_number}\n' +
+            verication_message.format(auth_url[0]))
+
+        print(ctx.author, 'Verification Started')
+        await logs.print(f'{ctx.author.mention} started Verification')
+
+        try:
+            authorization_code = await self.bot.wait_for(
```

Comment:
authorization_code is not the message, use content attribute to get the message
authorization_code  = authorization_code.content

Reply:
done